tools/wini/wini.py

```diff
@@ -80,6 +83,15 @@ def package_wandb_core(should_install, with_coverage):
     """Creates the wandb-core wheel, optionally installing it."""
     _build_wandb_core_artifacts(with_coverage=with_coverage)
 
+    if _CORE_WHEEL_DIR.exists():
```

Comment:
Maybe make it conditional of a flag?

Reply:
I'm thinking of "./core/dist" as being controlled by `wini` and assuming that any files there are generated by `wini`. If we're not comfortable with that, I could alternatively make a ".wini/" directory for this purpose and output the wheel there. What do you think?

Reply:
yeah i like this.
we could also use the cache directory as a parent dir?

Reply:
do we want to support  a case where i have multiple wheels? or we just rebuild every time since it just really fast?

Reply:
I don't know of a case where we'd want multiple wheels; maybe if we want to build once, then make changes and build again? But in that case, we'd also want to make `./wini package wandb-core` make sure it doesn't overwrite the previous wheel.

Reply:
yeah, i was thinking of a case where we want to compare different versions ...

Reply:
(so the alternative is always build and just install into 2 different envs), which is reasonable, given how fast the build takes

Reply:
updated wini to use a private `.wini/` directory